builder/image/Dockerfile

```diff
@@ -13,6 +13,10 @@ RUN curl -sSL -o /usr/local/bin/etcdctl https://s3-us-west-2.amazonaws.com/opdem
 RUN curl -sSL -o /usr/local/bin/confd https://s3-us-west-2.amazonaws.com/opdemand/confd-v0.5.0-json \
     && chmod +x /usr/local/bin/confd
 
+# install herokuish
+RUN curl -sSL https://github.com/aledbf/herokuish/releases/download/v0.1.2/herokuish_0.1.2_linux_x86_64.tgz \
```

Comment:
@aledbf could you send him a PR. I don't see any obvious issue with building multiple apps in the same container when we separate all dirs carefully. Somebody could kill the entire builder with a custom BP, but this is already a problem in our existing implementation.